src/test/java/core/basesyntax/service/strategy/OperationStrategyTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.ProductDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.FruitTransaction.Operation;
+import core.basesyntax.service.impl.ProductDaoImpl;
+import core.basesyntax.service.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.service.strategy.impl.PursheOperationHandler;
+import core.basesyntax.service.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.service.strategy.impl.SupplyOperationHandler;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class OperationStrategyTest {
+    private static ProductDao productDao;
+    private static OperationStrategy operationStrategy;
+
+    @BeforeAll
+    static void setUp() {
+        productDao = new ProductDaoImpl();
+        operationStrategy = new OperationStrategy(productDao);
+    }
+
+    @Test
+    void getOperation_getBalanceOperation_ok() {
+        OperationHandler expected = new BalanceOperationHandler(productDao);
```

Comment:
```suggestion
        Class<BalanceOperationHandler> expected = BalanceOperationHandler.class;
```
please fix in similar way in the other test methods.